spec/jobs/warm_entry_cache_job_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe WarmEntryCacheJob, type: :job do
+  before(:each) { ActiveJob::Base.queue_adapter = :test }
+
+  around do |example|
+    ClimateControl.modify(
+      CONTENTFUL_ENTRY_CACHING: "true"
+    ) do
+      example.run
+    end
+  end
+
+  describe "#perform_later" do
```

Comment:
```suggestion
  describe ".perform_later" do
```

It's a class method isn't it?